diff --git a/utils/loaders.py b/utils/loaders.py
index d630f26..f8f94ec 100644
--- a/utils/loaders.py
+++ b/utils/loaders.py
@@ -77,7 +77,7 @@ def set_random_seed(cfg):
 
 def load_config(config_file):
     with open(config_file) as f:
-        config = DotDict(yaml.load(f))
+        config = DotDict(yaml.load(f, Loader = yaml.FullLoader))
     if 'distributed' not in config:
         config['distributed'] = False
     return config
